practice_app/pages/api/getUpdatedPrice.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios'
+
+
+export default (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET': {
+        	// This is not important for now
+        	var productID = query.pid 
+        	// Creating a random product price
+        	var productPrice = 60
+        	// hold client's ip adress
+        	var clientIP = query.ip == null ? "" : query.ip
+
+        	//use two apis synchroniously
+	        Promise.all([
```

Comment:
Please use a code formatter in your code editor (for example Prettier) so that indentation is properly done.